src/operation/model_store_params.jl

```diff
@@ -0,0 +1,56 @@
+struct StoreModelRequirements
+    duals::Dict{ConstraintKey, Dict{String, Any}}
+    parameters::Dict{ParameterKey, Dict{String, Any}}
+    variables::Dict{VariableKey, Dict{String, Any}}
+end
+
+function StoreModelRequirements()
+    return StoreModelRequirements(
+        Dict{ConstraintKey, Dict{String, Any}}(),
+        Dict{ParameterKey, Dict{String, Any}}(),
+        Dict{VariableKey, Dict{String, Any}}(),
+    )
+end
+
+struct StoreModelParams
+    num_executions::Int
+    horizon::Int
```

Comment:
@daniel-thom we could do away with horizon since it is already in the container.

Reply:
> Nice consolidation. I have one ask: consistent use of store_model vs model_store.
> model_store_params.jl
> StoreModelParams
> model_store_init

Fixed in  2312d91